### PR TITLE
DDF-4350 Add internal/system hostname and port to the installer

### PR DESCRIPTION
**Pull request by vinamartin**
#### What does this PR do?
Adds internal hostname and port to the installer and reverts https://github.com/codice/ddf/pull/3841. 
Updates less because the titles are slightly longer. 
Should the behavior for ?dev=true remain the same?

#### Who is reviewing it? 
@mcalcote @oconnormi @austinsteffes 

#### Ask 2 committers to review/merge the PR and tag them here.
@brendan-hofmann
@clockard

#### How should this be tested?
Walk through the installer and make sure that updating the external and internal ports takes effect. 
Verify you can still log in after using ?dev=true

#### What are the relevant tickets?
[DDF-4350 Installer doesn't allow users to input internal/system hostname and port](https://codice.atlassian.net/browse/DDF-4350)

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/packages/ui/src/main/webapp/js/models/installer/Configuration.js [hunk 1]
```diff
@@ -79,12 +79,6 @@ define([
                 propertiesMap[model.get('key')] = model.get('value');
             });
 
-            var devMode = location.search.indexOf('dev=true') > -1 ? true : false;
```
Comment (vinamartin) [inline ui/packages/ui/src/main/webapp/js/models/installer/Configuration.js]: I've reverted this (https://github.com/codice/ddf/pull/3841) as it's easier to test the PR without this part, but I'm curious what people think the behavior should be if ?dev=true.

Comment (emmberk) [reply ui/packages/ui/src/main/webapp/js/models/installer/Configuration.js]: As long as you can still use the installer UI with `?dev=true`, I think it's fine to revert https://github.com/codice/ddf/pull/3841.

**Comment by vinamartin** [comment]
build now

**Comment by clockard** [comment]
Can you attach a screen shot of the new installer config page?

**Comment by vinamartin** [comment]
build now

**Comment by vinamartin** [comment]
build now

**Comment by ahoffer** [comment]
Heroed commit 9c38e743a89cef896d8a28390994404d4b015ca2.
**Test passed.**  Here are the details of what I tested it:

### Test  - Non-dev install process
#### Set-up:
* Create backup copy of keystore and truststore
* User CertNew script to upate keystore and truststore (cn=aaron.local)
* Move updated keystores to a new location
* Restore backups of original keystore and truststore
    
#### Preconditions:
* org.codice.ddf.system.hostname=localhost
* org.codice.ddf.system.httpsPort=8993
* org.codice.ddf.system.httpPort=8181
* org.codice.ddf.system.port=${org.codice.ddf.system.httpsPort}
    
#### Steps:
1.  Start DDF
2. Navigate to Admin UI
3. Login as "admin"
4. Start install
5. Set these values during installation:
   * internal hostname=aaron.local
   * internal https port=4141
   * internal http port=5151
9. Add updated keystore and trustore (passwords="changeit")
10. Restart system

##### Expected Result:
* org.codice.ddf.system.hostname=aaron.local
* org.codice.ddf.system.httpsPort=4141
* org.codice.ddf.system.httpPort=5151

##### Actual Result:
* org.codice.ddf.system.hostname=aaron.local
* org.codice.ddf.system.httpsPort=4141
* org.codice.ddf.system.httpPort=5151

####  Test Login:
* Navigate to admin page (https://aaron.local:4141/admin/)
* Login as admin/admin

#####  Expected Result:
* Successful login

##### Actual Result:
* Successful login (on second login attempt)

**Comment by vinamartin** [comment]
build now

**Pull request merged**